addons/edi_stock/models/edi_move_cancellation_report_document.py

```diff
@@ -20,6 +20,13 @@ class StockMove(models.Model):
         index=True,
     )
 
+    cancelled_by = fields.Many2one("res.users", string="Cancelled By", readonly=True)
```

Comment:
would rather this be named `cancelled_by_id` to conform to m2o naming, having `_id` suffix helps identify it is a relational field

Reply:
Also, unsure if this along with the `_action_cancel()` hook should / could live elsewhere, it seems like it could be a generic feature, but difficult as this repo likes to be standalone.

Reply:
I am finishing off this story. I think we should keep the `_action_cancel()` hook here as it is only used for EDI at the moment. If in the future if hss want to show the person who cancelled the picking on the pick then maybe we could refactor this and move it elsewhere?